lib/elements/dom-repeat.html

```diff
@@ -53,13 +53,13 @@
    *       is: 'employee-list',
    *       ready: function() {
    *         this.employees = [
-   *             {first: 'Bob', last: 'Smith'},
-   *             {first: 'Sally', last: 'Johnson'},
-   *             ...
+   *           {first: 'Bob', last: 'Smith'},
+   *           {first: 'Sally', last: 'Johnson'},
+   *           ...
    *         ];
    *       }
    *     });
-   *   < /script>
```

Comment:
Please leave the space in the `script`. Otherwise the `</script>` will break this module by ending the script early.